proto/wechaty/puppet.proto

```diff
@@ -93,7 +93,9 @@ service Puppet {
 
   rpc MessageContact (puppet.MessageContactRequest) returns (puppet.MessageContactResponse) {}
   rpc MessageFile (puppet.MessageFileRequest) returns (puppet.MessageFileResponse) {}
```

Comment:
Please add a comment with `@deprecated` warning to this deprecated method: `MessageFile`